backend/coins/eth/account.go

```diff
@@ -222,6 +224,23 @@ func (account *Account) update() error {
 		return err
 	}
 
+	// Nonce to be used for the next tx, fetched from the ETH node. It might be out of date due to
+	// latency, which is addressed below by using the locally stored nonce.
```

Comment:
The latency can also be not yet confirmed, right? If yes: `Since only confirmed transactions are considered, the nonce can be out-of-date, which is addressed …`

Reply:
No, pending ones are taken into account by using `PendingNonceAt` instead of `NonceAt`.